app/controllers/api/functions.php

```diff
@@ -1049,8 +1048,15 @@
             return $response->dynamic($execution, Response::MODEL_EXECUTION);
         }
 
-        /** Collect environment variables */
-        $vars = \array_merge($function->getAttribute('vars', []), [
+        $vars = [];
+
+        $variables = $function['vars'];
+
+        foreach ($variables as $variable) {
+            $vars[$variable['key']] = $variable['value'];
+        }
```

Comment:
```suggestion
        $variables = array_reduce($function['vars'] ?? [], function (array $carry, array $var) {
            $carry[$var['key']] = $var['value'];
            
            return $carry;
        }, []);
```

something like that makes it less noisy and not creates too many var/vars/variable variables :-D

Reply:
Addressed in all places